packages/suite/src/components/suite/SuiteLayout/index.tsx

```diff
@@ -89,11 +89,12 @@ const DefaultPaddings = styled.div`
     }
 `;
 
-const StyledGuidePanel = styled(GuidePanel)<{ open?: boolean }>`
+const StyledGuidePanel = styled(GuidePanel)<{ open?: boolean; isModalOpen?: boolean }>`
     height: 100%;
     width: ${variables.LAYOUT_SIZE.GUIDE_PANEL_WIDTH};
     flex: 0 0 ${variables.LAYOUT_SIZE.GUIDE_PANEL_WIDTH};
-    z-index: ${variables.Z_INDEX.GUIDE_PANEL};
+    z-index: ${props =>
+        props.isModalOpen ? variables.Z_INDEX.GUIDE_PANEL : variables.Z_INDEX.NAVIGATION_BAR - 1};
```

Comment:
What about two variables:
e.g. `GUIDE_PANEL` and `GUIDE_PANEL_MODAL`

Reply:
`GUIDE_PANEL_MODAL` makes no sence for me as there's no modal in guide. Should it be instead if `NAVIGATION_BAR - 1`?

Reply:
naming is hard, but I agree with the principle that Tomáš is referring to: define a separate variable for it.

there is no modal in guide, but there is "modal with guide". so something like `GUIDE_PANEL_BESIDE_MODAL`? (while modal is open and keep GUIDE_PANEL if there is no modal...)

Reply:
renamed in a7b4d11